lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -30,6 +30,43 @@ def _invert_angle(angle):
     return (180 - angle) % 360
 
 
+def get_dicom_wedge_info(beam_reference, field):
+
+    for cp in field.ControlPointSequence:
+        if cp.WedgePositionSequence[0].WedgePosition == "OUT":
+            wedge_MU = cp.CumulativeMetersetWeight
+            break
+
+    wedge_info = {
+        "wedge_type": field.WedgeSequence[0].WedgeType,
+        "wedge_angle": field.WedgeSequence[0].WedgeAngle,
+        "wedge_orienttation": field.WedgeSequence[0].WedgeOrientation,
+        "wedge_MU": wedge_MU * beam_reference.BeamMeterset,
+    }
+    return wedge_info
+
+
+def get_dicom_coll_info(field):
+
+    keys = {
+        "coll_x1": [0, 0],
+        "coll_x2": [0, 1],
+        "coll_y1": [1, 0],
+        "coll_y2": [1, 1],
+    }
+
+    colls = {}
+    for key, value in keys.items():
+        colls[key] = (
+            field.ControlPointSequence[0]
+            .BeamLimitingDevicePositionSequence[value[0]]
```

Comment:
The order here isn't always consistent. Instead, you need to look to a seperate field to see which index corresponds to which collimation device.

Reply:
Any ideas where I might be able to find that information off the top of your head? If not, I will do some searching!

Reply:
Not off the top of my head sorry 🙁 ... I do have code somewhere in PyMedPhys that deals with it though... Not available to find that at the moment though.